osgar/drivers/kloubak.py

```diff
@@ -99,7 +140,10 @@ def compute_desired_angle(desired_speed, desired_angular_speed):
     if abs(desired_angular_speed) < 0.000001:
         return 0.0
     radius = desired_speed/desired_angular_speed
-    return 2 * math.atan(CENTER_AXLE_DISTANCE / radius)
+    desired_angle = 2 * math.atan(CENTER_AXLE_DISTANCE / radius)
+    if abs(desired_angle) > MAX_JOIN_ANGLE:
+        return MAX_JOIN_ANGLE * desired_angle / abs(desired_angle) # return correct sign
```

Comment:
please put 2 spaces after the code (PEP8) to make it slightly more readable

Reply:
maybe `copysign` can be used here?
```
>>> a = 3
>>> math.copysign(-123.5)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: copysign expected 2 arguments, got 1
>>> math.copysign(a, -123.5)
-3.0
```
https://stackoverflow.com/questions/1986152/why-doesnt-python-have-a-sign-function

Reply:
Ok